subprojects/core/src/main/java/org/gradle/internal/service/scopes/GradleUserHomeScopeServices.java

```diff
@@ -302,12 +302,13 @@ public boolean isGeneratedByGradle(File file) {
             public void recordOutputs(Iterable<? extends FileSystemSnapshot> outputFileFingerprints) {
             }
         };
+        // TODO: Figure out how to get rid of origin scope id in snapshot outputs step
+        UniqueId fixedUniqueId = UniqueId.from("dhwwyv4tqrd43cbxmdsf24wquu");
```

Comment:
I didn't use a constant since it would be far from where we create the service and available to all other services in `GradleUserHomeScopeServices`. I prefer the reduced scope of a local variable. If you feel strongly, I can make it a constant.

Reply:
As we discussed on Slack, let's move the real build ID to `BuildTreeScopeServices`, and expose it via `UnitOfWork`.